src/core/varinfo.jl

```diff
@@ -27,25 +34,29 @@ cuid(vn::VarName) = (vn.csym, vn.sym, vn.indexing) # the uid which is only avail
 Base.getindex(graddict::Dict, vn::VarName) = graddict[uid(vn)]
```

Comment:
perhaps we can remove `Base.getindex(graddict::Dict, vn::VarName)` and `Base.setindex!(graddict::Dict, val, vn::VarName)` now